.circleci/config.yml

```diff
@@ -0,0 +1,43 @@
+version: 2
+jobs:
+  build:
+    docker:
+      - image: circleci/ruby:2.4.1-node-browsers
+      - image: circleci/postgres:9.4
+
+    steps:
+      - checkout
+      - restore_cache:
+          keys:
+          - v1-dependencies-{{ checksum "Gemfile.lock" }}
+
+      - run:
+          name: install dependencies
+          command: |
+            bundle install --jobs=4 --retry=3 --path vendor/bundle
+      - save_cache:
+          paths:
+            - ./venv
+          key: v1-dependencies-{{ checksum "Gemfile.lock" }}
+
+      # Database setup
+      - run: cp .circleci/database.yml config/database.yml
+      - run: cp .circleci/application.yml config/application.yml
+      - run: RAILS_ENV=test bundle exec rake db:create
+      - run: RAILS_ENV=test bundle exec rake db:migrate
+      - run: RAILS_ENV=test bundle exec rake db:schema:load
```

Comment:
Why run migrations and then load the schema? Shouldn't it be one of the two?

Reply:
I can remove the migrate one, I just copied it